pkg/apis/configuration/v1/kongingress_types.go

```diff
@@ -34,9 +34,15 @@ type KongIngress struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Upstream represents a virtual hostname and can be used to loadbalance
+	// incoming requests over multiple services (targets).
 	Upstream *KongIngressUpstream `json:"upstream,omitempty"`
-	Proxy    *KongIngressService  `json:"proxy,omitempty"`
-	Route    *KongIngressRoute    `json:"route,omitempty"`
+	// Proxy is the abstraction of each of your own upstream services.
+	Proxy *KongIngressService `json:"proxy,omitempty"`
```

Comment:
This field is perhaps a bit unfortunately named (honestly the whole `KongIngress` resource is in need of a renaissance as it were) and I'm not sure about this description. :thinking: 

A big part of what the `Proxy` field governs is Kong Gateway(/proxy) connection options for matching routes. Perhaps we could try this description?

```suggestion
	// Proxy defines additional connection options for the routes to be configured in the
	// Kong Gateway, e.g. `connection_timeout`, `retries`, e.t.c.
	Proxy *KongIngressService `json:"proxy,omitempty"`
```